### PR TITLE
Smartgunner fires slower and harness is removed. Sunder is a bit more.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Smartgunner harness no longer can be acquired unless your an FC or something.
Smartgun fires slower, but does the same sunder in a second.

## Why It's Good For The Game
idk people complain about sg
:cl:
balance: sg fires slower and no longer has a harness
:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -7. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -455,7 +455,7 @@
 	starting_attachment_types = list(/obj/item/attachable/stock/t42stock)
 	gun_skill_category = GUN_SKILL_HEAVY_WEAPONS
 	attachable_offset = list("muzzle_x" = 31, "muzzle_y" = 17,"rail_x" = 4, "rail_y" = 20, "under_x" = 16, "under_y" = 13, "stock_x" = 0, "stock_y" = 13)
-	fire_delay = 0.2 SECONDS
+	fire_delay = 0.25 SECONDS
```
Comment (GLACoding) [inline code/modules/projectiles/guns/rifles.dm:458]: This is the fire delay for the PFC LMG

Comment (Pariah919) [reply code/modules/projectiles/guns/rifles.dm]: don't code at 12pm when tired and trying to do and like me

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-8). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**